carp.studies.core/src/commonMain/kotlin/dk/cachet/carp/studies/infrastructure/versioning/RecruitmentServiceApiMigrator.kt

```diff
@@ -70,6 +70,9 @@ private val major1Minor2To3Migration =
                 }
             }
 
+            // Remove newly added 'roleAssignments' field from 'ParticipantGroupStatus'.
```

Comment:
Comment doesn't match code. It's also pretty self-documenting so maybe can be removed altogether and grouped with "name" removal under a comment similar to:

> Remove newly added fields from `ParticipantGroupStatus`.